storage/src/vespa/storage/visiting/visitor.cpp

```diff
@@ -637,7 +637,7 @@ Visitor::handleDocumentApiReply(mbus::Reply::UP reply,
 
     if (!reply->hasErrors()) {
         metrics.averageMessageSendTime[getLoadType()].addValue(
-                (message->getTimeRemaining() - message->getTimeRemainingNow()) / 1000.0);
+                (message->getTimeRemaining() - message->getTimeRemainingNow()).count() / 1000.0);
```

Comment:
we should have better conversion functions for duration to seconds/milliseconds/etc

Reply:
I will leave that to you to ponder about.
Leaving as is.